language/de.js

```diff
@@ -17,8 +17,40 @@ H5PEditor.language.core = {
   addFile: 'Datei hinzufügen',
   removeFile: 'Datei entfernen',
   confirmRemoval: 'Ganz sicher entfernen?',
+  browseFiles: "Dateien durchsuchen",
```

Comment:
Is this a test 🥲

Reply:
😅 Copy and paste (https://github.com/h5p/h5p-editor-php-library/blob/master/language/en.js#L20). That's what you get if you use github to do things ...